src/browserview/BrowserComponent.java

```diff
@@ -526,4 +526,12 @@ public void showContributors() {
 			}
 		});
 	}
+
+	public boolean isCurrentUrlIssue() {
+		if (driver.getCurrentUrl().matches("https://github.com/(.+)/(.+)/(issues|pull)/([0-9]+)")) {
+			return true;
+		}
+		return false;
+	}
+
```

Comment:
This method should go in `GitHubURL`.

Also, minor problem, but we should use `([^/]+)` instead of `(.+)`. That would reject urls like `https://github.com/////issues/1`.